pkg/sql/schemachanger/testdata/alter_table_add_column

```diff
@@ -2664,3 +2664,1454 @@ update progress of schema change job #1: "all stages completed"
 commit transaction #11
 notified job registry to adopt jobs: [2]
 # end PostCommitPhase
+
+test
+ALTER TABLE db.public.tbl ADD COLUMN m INT NOT NULL UNIQUE DEFAULT nextval('db.public.sq1')
```

Comment:
Honestly I'm starting to think that every new test case should go in a new file. What do you think?